content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -37,10 +37,10 @@ Refer to the following table for Windows operating system support in Kubernetes.
 
 | Kubernetes version | Windows Server LTSC releases | Windows Server SAC releases |
 | --- | --- | --- | --- |
-| *Kubernetes v1.17* | Windows Server 2019 | Windows Server ver 1809 |
-| *Kubernetes v1.18* | Windows Server 2019 | Windows Server ver 1809, Windows Server ver 1903, Windows Server ver 1909 |
+| *Kubernetes v1.18* | Windows Server 2019 | Windows Server ver 1909 |
 | *Kubernetes v1.19* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
 | *Kubernetes v1.20* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
+| *Kubernetes v1.21* | Windows Server 2019 | Windows Server ver 20H2, Windows Server ver 2004 |
```

Comment:
```suggestion
| *Kubernetes v1.21* | Windows Server 2019 | Windows Server ver 2004, Windows Server ver 20H2 |
```

Reply:
nit... but lets keep the older OS versions first in the list.

Reply:
I committed your suggestion.